app/services/concerns/xccdf/rule_groups.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving groups
+  module RuleGroups
+    extend ActiveSupport::Concern
+
+    included do
+      def save_rule_groups
+        @rule_groups ||= @op_rule_groups.map do |op_rule_group|
+          ::RuleGroup.from_openscap_parser(op_rule_group, benchmark_id: @benchmark&.id)
+        end
+
+        ::RuleGroup.import!(@rule_groups, ignore: true)
+
+        @rule_groups_with_parents = rule_group_parents
+        ::RuleGroup.import!(@rule_groups_with_parents, on_duplicate_key_update: {
+                              conflict_target: %i[ref_id benchmark_id],
+                              columns: :all
```

Comment:
You should not update `all` columns, please only specify the ones that are actually relevant here.